README.md

```diff
@@ -1,46 +1,93 @@
 The dart_style package defines an automatic, opinionated formatter for Dart
 code. It replaces the whitespace in your program with what it deems to be the
-best formatting for it. Resulting code should following the [Dart style guide][]
+best formatting for it. Resulting code should follow the [Dart style guide][]
 but, moreso, should look nice to most human readers, most of the time.
 
 [dart style guide]: https://www.dartlang.org/articles/style-guide/
 
-It handles indentation, inline whitespace and (by far the most difficult),
-intelligent line wrapping. It has no problems with nested collections, function
+The formatter handles indentation, inline whitespace and
+(by far the most difficult), intelligent line wrapping.
+It has no problems with nested collections, function
 expressions, long argument lists, or otherwise tricky code.
+Only white space is affected.
 
-## Usage
+To use the formatter, you can either run the **dartfmt** command or use the
+dart_style API.
 
-The package exposes a simple command-line wrapper around the core formatting
-library. The easiest way to invoke it is to [globally activate][] the package
-and let pub put its executable on your path:
+IDEs and editors that support Dart usually provide easy ways to run the
+formatter. For example, in WebStorm you can right-click a .dart file
+and then choose **Reformat with Dart Style**.
+
+**Note:** dartfmt is safe to run automatically. We encourage you to add
+it to your build scripts.
+
+The following example shows a complex expression, with most white
+space removed, and the code that results when using dartfmt.
```

Comment:
Sentence fragments like this are considered tech writing badness. Since this isn't on dartlang, I guess we can be more flexible, though. Kathy would never allow it on dartlang.  :)